scripts/cmux-events-stress.py

```
@@ -0,0 +1,653 @@
+                pass
+            self.sock = None
+
+    def write_request(self, request: dict[str, Any]) -> None:
+        if self.sock is None:
+            raise StressFailure("socket is not connected")
+        line = json.dumps(request, separators=(",", ":"), sort_keys=True) + "\n"
+        self.sock.sendall(line.encode("utf-8"))
+
+    def read_frame(self) -> dict[str, Any]:
+        if self.reader is None:
+            raise StressFailure("socket reader is not connected")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)